Modules/core/Controller/CorespaceaccessController.php

```diff
@@ -202,12 +202,19 @@ public function useraddAction($id_space){
         $form->setValidationButton(CoreTranslator::Ok($lang), "corespaceaccessuseradd/".$id_space);
 
         if ($form->check()) {
-
             $modelCoreUser = new CoreUser();
+            $canEditUser = true;
+            if ($modelCoreUser->isLogin($this->request->getParameter('login'))) {
+                $canEditUser = false;
+                throw new PfmException(CoreTranslator::LoginAlreadyExists($lang), 403);
```

Comment:
why throwing an exception here but not in email case?
both should raise an exception or display flash error

Reply:
It was a PfmException at first for login. I think you put it there. But I prefer flash messages, more user friendly. I push back with a flash message